CI-CD/AzureDevOps/azure-pipelines_dotnet_sbom.yml

```diff
@@ -39,7 +39,7 @@ steps:
 - script: |
     export WS_PROJECTTOKEN=$(jq -r '.projects | .[] | .projectToken' ./whitesource/scanProjectDetails.json)
     pip install ws-sbom-generator
-    ws_sbom_generator -u $WS_USERKEY -k $WS_APIKEY -s $WS_PROJECTTOKEN -a $WS_URL -t tv -o ./whitesource
+    ws_sbom_generator -u $WS_USERKEY -k $WS_APIKEY -s $WS_PROJECTTOKEN -a $WS_URL -t tv -o ./Mend
```

Comment:
Is this folder name supported? Is it the (still) hard-coded ./whitesource folder the UA creates, or is this an arbitrary name in this context?

Reply:
@lukebrogan-mend this should remain /whitesource as it is the folder for the unified agent scan...which I believe is still whitesource